syntax = "proto3";

import "player.proto";

message OptimizerRequest {
    repeated string player_id_locks = 1;
    
    float randomness = 2;

    int32 num_lineups = 3;

    bool stack = 4;
}

message OptimizerResponse {
    repeated Lineup lineups = 1;
}

message GetPlayersRequest {}

message GetPlayersResponse {
    Players players = 1;
}